packages/trip-form/src/TripOptions/OptionButton.tsx

```diff
@@ -5,14 +5,15 @@ import * as S from "./styled";
 // eslint-disable-next-line prettier/prettier
 import type { ButtonProps } from "./Checkbox"
 
-export default function OptionButton({ onClick, label, children, className, selected, checked }: {
+export default function OptionButton({ onClick, label, children, className, selected, checked, disabled }: {
```

Comment:
Deserves prop sorting.

Reply:
Sorted in c58e1ba1b18a029d706b79ede998dd3b486e1d21